image/utils/meta_image.go

```diff
@@ -125,3 +125,19 @@ func SetImageMetadata(metadata ImageMetadata) error {
 	}
 	return nil
 }
+
+func GetNewImageMetadata(imageName string) (imageMetadata ImageMetadata, ok bool) {
```

Comment:
I am wondering if it returns an error is better. 
If you return a bool value, it could carry little information there. But an error could carry more. It is useful for user to get noticed of the detailed error reason.